angular/test/README.md

```diff
@@ -2,6 +2,18 @@
 
 Ionic Framework supports multiple versions of Angular. As a result, we need to verify that Ionic works correctly with each of these Angular versions.
 
+## Syncing Local Changes
+
+The Angular test app supports syncing your locally built changes for validation.
+
+1. Build the `@ionic/core` and `@ionic/angular` projects using `npm run build`.
```

Comment:
Should we instead write something like:

> Navigate to the `core` and `packages/angular` directories and build them using `npm run build`

I think it might be confusing if people think they're expected to deploy an npm package first.

Reply:
Also - don't they need to build `packages/angular-router` too?

Reply:
there's no `packages/angular-router` package

Reply:
Updated wording

Reply:
Derp I was thinking of react and vue